File: MDEV1001-M2023-ICE5/Android/app/src/main/java/com/kanwarabhijitsingh/ice5/Movie.kt
package com.kanwarabhijitsingh.ice5

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class Movie(
	@PrimaryKey(autoGenerate = true) val id: Int = 0,
	val title: String,
	val studio: String,
	val genres: String,
	val directors: String,
	val writers: String,
	val actors: String,
	val year: Int,
	val length: Int,
	val shortDescription: String?,
	val mpaRating: String,
	val criticsRating: Double
)

File: MDEV1001-M2023-ICE9/iOS/ICE9/ICE9/Movie.swift
//
//  Movie.swift
//  ICE9
//
//  Created by Abhijit Singh on 22/07/23.
//

struct Movie: Codable
{
    let _id: String
    let movieID: String?
    let title: String
    let studio: String
    let genres: [String]
    let directors: [String]
    let writers: [String]
    let actors: [String]
    let year: Int
    let length: Int
    let shortDescription: String
    let mpaRating: String
    let criticsRating: Double
}
